### PR TITLE
KAA-1169 Style fix according to the QA review

### PR DESCRIPTION
**Pull request by forGGe**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ sort_idx: 30
 
 This guide explains how to build applications for Texas Instruments CC3200 LaunchPad (further, CC3200) based on the Kaa C SDK.
 
-# Installing necessary components for Linux (Ubuntu)
+# installation steps for Linux (Ubuntu)
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: I meant to change it in the sentence below.

**Comment by Polinaa** [comment]
LGTM

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ This guide explains how to build applications for Texas Instruments CC3200 Launc
 
 # Installing necessary components for Linux (Ubuntu)
 
-Before building the C SDK for the CC3200 platform on Linux, you need to perform the following installation.
+Before building the C SDK for the CC3200 platform on Linux, you need to perform the installation steps.
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: "the next installation steps" or "the following installation steps"?

**Pull request merged**